components/QuakeButton.js

```diff
@@ -3,14 +3,39 @@ import { Alert, AppRegistry, Button, StyleSheet, View, Text } from 'react-native
 
 // Button Element
 export default class QuakeButton extends Component {
-
   constructor(props) {
     super(props);
     {/* make `this` available to `onPressQuake` */}
     this._onPressButtonQuake = this._onPressButtonQuake.bind(this);
   }
 
   _onPressButtonQuake() {
+  
+    // Getting location and timestamp,sending it to server
+    navigator.geolocation.getCurrentPosition(
+      (pos) => {
+        var crd = pos.coords;
+        fetch('http://10.190.54.186:8000/reports/', {
```

Comment:
esta dirección debería ser configurable: variable de entorno o algo así
https://medium.com/differential/managing-configuration-in-react-native-cd2dfb5e6f7b